nixpkgs_review/github.py

```diff
@@ -32,12 +32,28 @@ def get(self, path: str) -> Any:
     def post(self, path: str, data: Dict[str, str]) -> Any:
         return self._request(path, "POST", data)
 
+    def put(self, path: str) -> Any:
+        return self._request(path, "PUT")
+
     def issue_comment(self, pr: int, msg: str) -> Any:
         "Post a comment on a PR with nixpkgs-review report"
+        print(f"Posting result comment on PR {pr}")
         return self.post(
             f"/repos/NixOS/nixpkgs/issues/{pr}/comments", data=dict(body=msg)
         )
 
+    def pr_approve(self, pr: int) -> Any:
+        "Approve a PR"
+        print(f"Approving PR {pr}")
+        return self.post(
+            f"/repos/NixOS/nixpkgs/pulls/{pr}/reviews", data=dict(event="APPROVE"),
+        )
+
+    def pr_merge(self, pr: int) -> Any:
+        "Merge a PR. Requires maintainer access to NixPkgs"
+        print(f"Merging PR {pr}")
+        return self.put(f"/repos/NixOS/nixpkgs/pulls/{pr}/merge")
```

Comment:
This can be done in a later PR, but I like to do a `rebase and merge`. (makes the commit history a lot nicer)

Essentially need:
```
'{ "merge_method": "rebase"}'
```
as the body.